plugins/XmlMaterialProfile/product_to_id.json

```diff
@@ -12,5 +12,9 @@
     "Ultimaker Original": "ultimaker_original",
     "Ultimaker Original+": "ultimaker_original_plus",
     "Ultimaker Original Dual Extrusion": "ultimaker_original_dual",
-    "IMADE3D JellyBOX": "imade3d_jellybox"
+    "IMADE3D JellyBOX": "imade3d_jellybox",
+    "Strateo3D": "strateo3d",
+    "Strateo3D_IDEX420": "strateo3d_IDEX420",
+    "Strateo3D_IDEX420_Duplicate": "strateo3d_IDEX420_duplicate",
```

Comment:
On second thought, none of these changes are needed. The file name is what defines the ID, not the name.